arch/xtensa/Kconfig

```diff
@@ -91,6 +91,11 @@ config XTENSA_IMEM_REGION_SIZE
 	range 0x2000 0x28000
   default 0x28000
 
+config XTENSA_IMEM_PROCFS
+  bool "Internal memory PROCFS support"
+  default n
+	depends on XTENSA_USE_SEPERATE_IMEM && !DISABLE_MOUNTPOINT && FS_PROCFS && FS_PROCFS_REGISTER
```

Comment:
please align "depends on" below "default n"

Reply:
I think this is just a Github glitch.  You can see the file here: https://github.com/apache/incubator-nuttx/blob/7b1234bb38379c5ce28c3ffefa82c4f6b4b42e0d/arch/xtensa/Kconfig